src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -11,3 +11,9 @@
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: project.description }
+
+  - if policy(project).update? && feature_enabled?(:responsive_ux)
```

Comment:
It's not needed to check for the `responsive_ux` feature flag since we're removing this kind of code right now.

```suggestion
  - if policy(project).update?
```

Reply:
Okay